analyzer/runtime/runtime.go

```diff
@@ -135,6 +135,12 @@ func (m *processor) FinalizeFastSync(ctx context.Context, lastFastSyncHeight int
 	m.logger.Info("recomputing number of txs for every account")
 	batch.Queue(queries.RuntimeAccountNumTxsRecompute, m.runtime, lastFastSyncHeight)
 
+	m.logger.Info("recomputing total_sent for every account")
+	batch.Queue(queries.RuntimeAccountTotalSentRecompute, m.runtime, lastFastSyncHeight)
```

Comment:
Did you maybe test how long do these two added queries take on a live db? 

Since these are run only once after fast-sync completes it doesn't really matter all that much, but would be interesting to know the rough running time.

Reply:
Each takes ~10-15 seconds on sapphire staging mainnet, though I'd expect them to take longer on emerald.